src/tcgwars/logic/impl/gen4/CallOfLegends.groovy

```diff
@@ -1151,427 +579,179 @@ public enum CallOfLegends implements LogicCardInfo {
-
-        };
+        return copy(Undaunted.MISDREAVUS_57, this);
       case PHANPY_66:
         return basic (this, hp:HP060, type:FIGHTING, retreatCost:2) {
           weakness W
           resistance L, MINUS20
           pokeBody "Ultra-Thick Skin", {
             text "As long as Phanpy has Energy attached to it, any damage done to Phanpy by attacks is reduced by 10 ."
-            delayedA {
+            delayed {
+              before APPLY_ATTACK_DAMAGES, {
+                bg.dm().each {if(it.to==self && it.from.owner==self.owner.opposite && it.notZero && it.notNoEffect) {
+                  bc "$thisAbility -10"
+                  it.dmg-=hp(10)
+                }}
+              }
             }
           }
           move "Rock Smash", {
             text "20+ damage. Flip a coin. If heads, this attack does 20 damage plus 10 more damage."
             energyCost F, C
-            attackRequirement {}
             onAttack {
               damage 20
-              flip { damage 10 }
+              flip {
+                damage 10
+              }
             }
           }
 
         };
       case PIDGEY_67:
-        return basic (this, hp:HP050, type:COLORLESS, retreatCost:1) {
-          weakness L
-          resistance F, MINUS20
-          move "Messenger", {
-            text "Search your deck for a Pokémon, show it to your opponent, and put it into your hand. Shuffle Pidgey and all cards attached to it back into your deck."
-            energyCost C
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-          move "Glide", {
-            text "10 damage. "
-            energyCost C
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-
-        };
+        return copy(Triumphant.PIDGEY_71, this);
       case PINECO_68:
-        return basic (this, hp:HP060, type:GRASS, retreatCost:2) {
-          weakness R
-          move "Focus Energy", {
-            text "During your next turn, Pineco’s Surprise Attack attack’s base damage is 80."
-            energyCost C
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-          move "Surprise Attack", {
-            text "40 damage. Flip a coin. If tails, this attack does nothing."
-            energyCost G, C
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-
-        };
+        return copy(Undaunted.PINECO_63, this);
       case RELICANTH_69:
         return basic (this, hp:HP080, type:WATER, retreatCost:2) {
           weakness G
           move "Prehistoric Wisdom", {
             text "Choose a card from your hand and put it in the Lost Zone. Then, draw 3 cards."
             energyCost C
-            attackRequirement {}
+            attackRequirement {
+              assert my.deck|my.hand : "You have no cards in your hand or deck!"
```

Comment:
Missed this as well.

```suggestion
              assert my.deck || my.hand : "You have no cards in your hand or deck!"
```
I don't think a bitwise or would work here, best to use the logical or.